gym_minigrid/envs/crossing.py

```diff
@@ -30,7 +32,7 @@ def _gen_grid(self, width, height):
         self.grid.wall_rect(0, 0, width, height)
 
         # Place the agent in the top-left corner
-        self.agent_pos = (1, 1)
+        self.agent_pos = np.array((1, 1))
```

Comment:
This seems like an important type change from the old. Is there a reason for this?

Reply:
When given an initial `agent_pos` the type is `tuple`, but after the first step `agent_pos` is of type `numpy`. We should be consistent with the type.